Add SETEACL token to authmate (#353)
nspcc-dev/neofs-s3-gw: docs/authmate.md

With session token, there is 3 options: 
* append `--session-token` parameter with your custom rules in json format (as a string or file path, see an example below)

**NB!** To create buckets in NeoFS it's necessary to have session tokens with `PUT` and `SETEACL` permissions, that's why 

> the authmate will create a `SETEACL` session token automatically if a user wants to put a token with `PUT`  with wildcard.

Reviewer: ```suggestion
the authmate will create a `SETEACL` session token automatically if a user specified token rule with `PUT` and forgot rule with `SETEACL`.
```
